examples/hackernews-node-ts/src/script.ts

```diff
@@ -0,0 +1,27 @@
+// 1
+import { PrismaClient } from '@prisma/client'
+
+// 2
+const prisma = new PrismaClient()
+
+// 3
+async function main() {
+  await prisma.link.create({
+    data: {
+      description: 'Fullstack tutorial for GraphQL',
+      url: 'www.howtographql.com',
+    },
+  })
+  const allLinks = await prisma.link.findMany()
+  console.log(allLinks)
+}
+
+// 4
+main()
+  .catch((e) => {
+    throw e
```

Comment:
In case rethrowing error you can simple remove `catch` chain

Reply:
Maybe just process.exit(1) ???